packages/Header/Header.vue

```diff
@@ -1,14 +1,23 @@
 <template>
-  <div class="header">
-    <div class="container header-box">
-      <div class="title-box" :class="{ w50: !!image }">
-        <h1>{{ title }}</h1>
-        <slot>
-          <p>{{ description }}</p>
+  <div id="header">
+    <div v-if="image || $slots.image" class="container header-box">
```

Comment:
Do we use this slot `image`? If not then we could simplify and remove it

Reply:
Yes we use this slot to replace the image by something else. Example we use it for the details page of a contribution to replace the image by an embed card.